draft-ietf-oauth-identity-chaining.md

```diff
@@ -438,6 +438,16 @@ The flow contains the following steps:
 
 (G) The client in Domain A uses the received access token to access the protected resource in Domain B.
 
+## Delegated Key Binding
+
+In some environments, there is a need to bind the access token issued by the Authorization Server in Domain B to a private key held by the Resource Server in Domain A. This is so that the Resource Server in Domain B can verify the proof of possession of the private key of the client in Domain A when the client in Domain A presents the token to the Resource Server in Domain B. Any application in Domain A may act as a client, including applications that are resource servers in Domain A and need to access resource servers in Domain B in order to complete a request.
```

Comment:
to a private key held by the Resource Server in Domain A -> to a private key held by a Client in Domain A